docs/sources/source/nlp/dialogue_state_tracking.rst

```diff
@@ -264,3 +263,124 @@ References
     :style: plain
     :labelprefix: NLP-DST
     :keyprefix: nlp-dst-
+
+
+
+SGD Tutorial
+============
+
+
+Introduction
+------------
+
+One of the main building blocks of a task-oriented dialogue system that can book a table in a restaurant or order food for you is a Dialogue State Tracker (DST).
```

Comment:
Task-oriented dialogue system is a conversational system that is able to perform a conversation with a user and provide task- (or domain-)specific information. For example, it can book a table in a restaurant or buy a train ticket.
One of the main building blocks of a task-oriented dialogue system is a Dialogue State Tracker (DST).

Reply:
fixed, thanks